src/main/java/com/scrive/pdftools/tools/utils/S3ForLambda.java

```diff
@@ -0,0 +1,94 @@
+package com.scrive.pdftools.tools.utils;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.*;
+
+import java.io.*;
+import java.util.Optional;
+
+public class S3ForLambda {
+
+    private static final String ENV_PARAM_ACCESS_KEY = "access_key";
+    private static final String ENV_PARAM_SECRET_KEY = "secret_key";
+    private static final String ENV_PARAM_BUCKET = "bucket";
+    private static final String ENV_PARAM_FAKES3_HOST = "fakes3_host";
+    private static final String ENV_PARAM_FAKES3_PORT = "fakes3_port";
+    private static final Regions S3_REGION = Regions.EU_WEST_1;
+
+
+    public static String fromStreamToString(InputStream inputStream) throws IOException {
```

Comment:
`IOUtils.toString(inputStream, encoding)`

Reply:
Yes, fixing.

Reply:
Yes. Much nicer with this.